scania-digital-design-system/sdds
# feat(storybook): adding Scania logotype

**Describe pull-request**  
Adding Scania logotype to our Storybook

**How to test**  
1. Go to Storybook
2. Check if instead of the Storybook logo in the left menu it is Scania logotype that shows up
